### ISSUE TITLE
How come the setup is so complicated?

### ISSUE THREAD

**Issue by ghost** [Botangle/dayoneold#2]
I was hoping to use Botangle, or atleast check it out, but it seems so complicated to set up, and I need to download a bunch of stuff, how come it can't just be one program I download and open?

**Comment by rhlkmth** [comment]
This app isn't practical. Don't expect this to work.

**Comment by HedonieBrazil** [comment]
Yeah it seems it was interesting stuff. please make the program accessible again http://web.archive.org/web/20140710161410/https://www.botangle.com/
